packages/blocks/src/BrandLogos/BrandLogos.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FC } from 'react';
+import {
+  Flex,
+  Grid,
+  Image,
+  Text,
+  TextPairing,
+  TextPairingVariant,
+  textPairingVariants,
+  TextVariant,
+  textVariants,
+} from '@cmpsr/components';
+
+import { BrandLogosProps } from './types';
+
+export const BrandLogos: FC<BrandLogosProps> = ({
+  backgroundColor = 'background-page',
+  title,
+  description,
+  logos,
+  textVariant,
+}) => (
+  <Flex
+    data-testid="brand-logos"
```

Comment:
do we really need this?, I don't like having _test code_ in production, and if we really do, we should follow the `cmpsr.component.child-component` nomenclature